FreeScape.Engine/Config/TileSet/TileSetTile.cs

```diff
@@ -12,6 +12,6 @@ public class TileSetTile
         public string Image { get; set; }
         public float ImageWidth { get; set; }
         public float ImageHeight { get; set; }
-        public List<AnimationFrame> Animation { get; set; }
+        public List<AnimationFrameInfo> Animation { get; set; }
```

Comment:
I should have named this to have "Tiled" in the name (and the other tiled types) to prevent confusion. Maybe extract tiled specific impls